doc/how-to/vshard_quick.rst

```diff
@@ -1,160 +1,525 @@
+    ..  literalinclude:: /code_snippets/snippets/sharding/instances.enabled/sharded_cluster/router.lua
+        :language: lua
+        :start-at: function insert_data
+        :dedent:
+
+The resulting ``router.lua`` file should look as follows:
+
+..  literalinclude:: /code_snippets/snippets/sharding/instances.enabled/sharded_cluster/router.lua
+    :language: lua
+    :dedent:
+
+
+
+..  _vshard-quick-start-build-settings:
+
+Configuring build settings
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Open the ``sharded_cluster-scm-1.rockspec`` file and add the following content:
+
+..  literalinclude:: /code_snippets/snippets/sharding/instances.enabled/sharded_cluster/sharded_cluster-scm-1.rockspec
+    :language: none
+    :dedent:
+
+The ``dependencies`` section includes the specified version of the ``vshard`` module.
+To install dependencies, you need to :ref:`build the application <vshard-quick-start-building-app>`.
+
+
+.. _vshard-quick-start-building-app:
+
+Building the application
+------------------------
+
+In the terminal, open a directory where the :ref:`tt environment is created <vshard-quick-start-creating-app>`.
+Then, execute the ``tt build`` command:
+
+.. code-block:: console
+
+    $ tt build sharded_cluster
+       • Running rocks make
+    No existing manifest. Attempting to rebuild...
+       • Application was successfully built
+
+This installs the ``vshard`` dependency defined in the :ref:`*.rockspec <vshard-quick-start-build-settings>` file to the ``.rocks`` directory.
+
+
+
+..  _vshard-quick-start-working-cluster:
+
+Working with the cluster
+------------------------
+
+.. _vshard-quick-start-working-starting-instances:
+
+Starting instances
+~~~~~~~~~~~~~~~~~~
+
+To start all instances in the cluster, execute the ``tt start`` command:
+
+.. code-block:: console
+
+    $ tt start sharded_cluster
+       • Starting an instance [sharded_cluster:storage-a-001]...
+       • Starting an instance [sharded_cluster:storage-a-002]...
+       • Starting an instance [sharded_cluster:storage-b-001]...
+       • Starting an instance [sharded_cluster:storage-b-002]...
+       • Starting an instance [sharded_cluster:router-a-001]...
+
+
+.. _vshard-quick-start-working-bootstrap:
+
+Bootstrapping a cluster
+~~~~~~~~~~~~~~~~~~~~~~~
+
+To bootstrap the cluster, follow the steps below:
+
+1.  Connect to the router instance using ``tt connect``:
+
+    ..  code-block:: console
+
+        $ tt connect sharded_cluster:router-a-001
+           • Connecting to the instance...
+           • Connected to sharded_cluster:router-a-001
+
+2.  Call :ref:`vshard.router.bootstrap() <router_api-bootstrap>` to perform the initial cluster bootstrap:
+
+    ..  code-block:: console
+
+        sharded_cluster:router-a-001> vshard.router.bootstrap()
+        ---
+        - true
+        ...
+
+
+.. _vshard-quick-start-working-status:
+
+Checking status
+~~~~~~~~~~~~~~~
+
+To check the cluster's status, execute :ref:`vshard.router.info() <router_api-info>` on the router:
```

Comment:
Any hints what to look at? Now it's a big unfamiliar console output that hardly helps newcomers.